Introduction to xemc3.ipynb

```diff
@@ -0,0 +1,3171 @@
+    "def plot_emc3grid(ip):\n",
+    "    plt.figure(figsize=(15,15))\n",
+    "    ds.emc3.plot_Rz(None, phi=ds.coords['phi_bounds'][ip][0])\n",
+    "ip = widgets.IntSlider(min = 0, max = 35, value = 0)\n",
+    "interact(plot_emc3grid, ip = ip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "979dff4b",
+   "metadata": {},
+   "source": [
+    "## Grid with boundaries\n",
+    "\n",
+    "Interactive plot of the grid, here you can use the ip slider to iterate through all toroidal slices(all phi angles),\n",
+    "the rmin and rmax to set the boundaries in r direction, and the zmin and zmax to set the boundaries in z direction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "7d5cc869",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "38199524515a47e6b8bae1f54e38d861",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "interactive(children=(IntSlider(value=0, description='ip', max=35), FloatSlider(value=4.3563466449999995, desc…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<function __main__.plot_emc3grid(ip, rmin, rmax, zmin, zmax)>"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# find boundaries of the grid\n",
+    "rmin = np.min(np.asarray(R))\n",
+    "rmax = np.max(np.asarray(R))\n",
+    "zmin = np.min(np.asarray(z))\n",
+    "zmax = np.max(np.asarray(z))\n",
+    "\n",
+    "import ipywidgets as widgets\n",
+    "from ipywidgets import interact, interactive, fixed, interact_manual\n",
+    "def plot_emc3grid(ip, rmin, rmax, zmin, zmax):\n",
+    "    plt.figure(figsize=(100,100))\n",
+    "    ds.emc3.plot_Rz('Te', phi=ds.coords['phi_bounds'][ip][0], Rmin = rmin, Rmax = rmax, zmin = zmin, zmax = zmax)\n",
+    "ip = widgets.IntSlider(min = 0, max = 35, value = 0)\n",
+    "rmin_slider = widgets.FloatSlider(min = rmin, max = rmax, value = rmin, readout_format='.9f')\n",
+    "rmax_slider = widgets.FloatSlider(min = rmin, max = rmax, value =rmax, readout_format='.9f')\n",
+    "zmin_slider = widgets.FloatSlider(min = zmin, max = zmax, value = zmin, readout_format='.9f')\n",
+    "zmax_slider = widgets.FloatSlider(min = zmin, max = zmax, value = zmax, readout_format='.9f')\n",
+    "interact(plot_emc3grid, ip = ip, rmin = rmin_slider, rmax = rmax_slider, zmin = zmin_slider, zmax = zmax_slider)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "id": "cdb7d2f0",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[91] [120] [94] [0]\n"
+     ]
+    }
+   ],
+   "source": [
+    "idx_zmin = np.where(np.isclose(-0.77468, np.asarray(z), rtol = 1e-6) == True)[0]\n",
```

Comment:
not sure what you are trying to do here.

`ds.emc3.evaluate_at_...` would probably be in general the nicer approach